server/helpers/wowCommunityApi.js

```diff
@@ -38,33 +39,145 @@ const get = (url, metricLabels) => {
     },
     onFailure: err => {
```

Comment:
Please add a check for the error that is thrown when the access token is expired, and automatically refresh the access token then.

Reply:
I'm trying to catch expired tokens by checking expired time when getting it, granted that could still allow some to slip through. Just made the change to renew it again on a 401 failure